ui/perfherder/alerts/AlertTable.jsx

```diff
@@ -349,12 +355,21 @@ export default class AlertTable extends React.Component {
                         onChangeSort={this.onChangeSort}
                       />
                     </th>
-                    <th className="align-bottom">
+                    <th className="align-bottom text-nowrap">
```

Comment:
`text-nowrap` is not needed on the Platform header. Better move it on Debug Tools header